<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QQ中控</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>模拟器</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="6,2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="table_emulator">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">gridline-color: rgb(255, 0, 0)</string>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideRight</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::NoPen</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>15</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>15</number>
            </attribute>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>索引</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>标题</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>状态</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>脚本描述</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0" columnstretch="0,0,0,0,0">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>10</number>
              </property>
              <property name="verticalSpacing">
               <number>0</number>
              </property>
              <item row="2" column="0">
               <widget class="QPushButton" name="btn_sort_window">
                <property name="text">
                 <string>排列窗口</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btn_refresh_table">
                <property name="text">
                 <string>刷新列表</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QPushButton" name="btn_select_all">
                <property name="text">
                 <string>全选</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="btn_select_cancel">
                <property name="text">
                 <string>取消选择</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="btn_close_ld">
                <property name="text">
                 <string>关闭模拟器</string>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="btn_select_invert">
                <property name="text">
                 <string>反选</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="btn_open_ld">
                <property name="text">
                 <string>打开模拟器</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QPushButton" name="btn_restart_ld">
                <property name="text">
                 <string>重启模拟器</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="1">
               <widget class="QPushButton" name="btn_copy_qq_number">
                <property name="text">
                 <string>复制QQ</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="btn_copy_guid">
                <property name="text">
                 <string>复制GUID</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btn_import_qq">
                <property name="text">
                 <string>批量导入</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="btn_copy_password">
                <property name="text">
                 <string>复制密码</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>355</width>
               <height>395</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;线程选择&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmb_thread">
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <item>
                   <property name="text">
                    <string>多线程</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>单线程</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTabWidget" name="tabWidget_2">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="tab_3">
                 <attribute name="title">
                  <string>登录功能</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout" rowstretch="3,0,0" columnstretch="0,0,0,0">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QCheckBox" name="checkbox_set_guid">
                      <property name="text">
                       <string>设置GUID</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QCheckBox" name="checkbox_input_data">
                      <property name="text">
                       <string>输入账号密码</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QCheckBox" name="checkbox_login_qq">
                      <property name="text">
                       <string>登录QQ</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="3">
                     <widget class="QCheckBox" name="checkbox_quit_login">
                      <property name="text">
                       <string>退出登录</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="3">
                     <widget class="QCheckBox" name="checkbox_auto_enable">
                      <property name="text">
                       <string>手动模式</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QCheckBox" name="checkbox_qq_setting">
                      <property name="text">
                       <string>QQ设置</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QCheckBox" name="checkbox_switch_login">
                      <property name="text">
                       <string>切换账号</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QCheckBox" name="checkbox_debug">
                      <property name="text">
                       <string>调试功能</string>
                      </property>
                      <property name="checkable">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QCheckBox" name="checkbox_clear_verfity">
                      <property name="text">
                       <string>清除验证</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="tab_4">
                 <attribute name="title">
                  <string>功能页2</string>
                 </attribute>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <widget class="QCheckBox" name="checkbox_jump_scan_page">
                    <property name="text">
                     <string>跳转扫码页</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QCheckBox" name="checkbox_scan_login">
                    <property name="text">
                     <string>扫码登录</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>参数设置</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>上号文件编号:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="input_number_index">
                   <property name="text">
                    <string>1</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="1">
                 <widget class="QPushButton" name="btn_stop_script">
                  <property name="text">
                   <string>停止脚本</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="btn_startup_script">
                  <property name="text">
                   <string>启动脚本</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>控制台</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>日志输出</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextBrowser" name="text_console">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
